### ISSUE TITLE
clang_parser tests fail on host machine, succeed inside containers

### ISSUE THREAD

**Issue by mmarchini** [iovisor/bpftrace#569]
Ok, this has annoyed me for a while. A few clang_parser tests are failing constantly on my host machine for months. I ignored because I thought it was something in my environment, and since I often use containers anyway it didn't bother me much. I switched to a new computer and I'm building and testing more on the host now, and the problem persists.

These are the failing tests:

```
[  FAILED  ] clang_parser.integers
[  FAILED  ] clang_parser.c_union
[  FAILED  ] clang_parser.integer_ptr
[  FAILED  ] clang_parser.string_ptr
[  FAILED  ] clang_parser.string_array
[  FAILED  ] clang_parser.nested_struct_named
[  FAILED  ] clang_parser.nested_struct_ptr_named
[  FAILED  ] clang_parser.nested_struct_anon
```

Looking at clang_parser.integers, the error is:

```
[ RUN      ] clang_parser.integers
/home/mmarchini/workspace/iovisor/bpftrace/tests/clang_parser.cpp:30: Failure
Expected equality of these values:
  structs.size()
    Which is: 3
  1U
    Which is: 1
[  FAILED  ] clang_parser.integers (46 ms)
```

So the test expects only one struct to be defined, but on my machine it has 3 for some reason ¯\_(ツ)_/¯

I added some printfs, and those are the structs:

```
Foo
ftrace_branch_data
ftrace_likely_data
```

Foo is supposed to be there, but those ftrace_* thingys are not.

Anyone else experiencing this?

**Comment by mmarchini** [comment]
Environment:

```
$ uname -a
Linux laul-mmarchini 4.15.0-47-generic #50-Ubuntu SMP Wed Mar 13 10:44:52 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux
$ lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 18.04.2 LTS
Release:        18.04
Codename:       bionic
```

libbcc from iovisor's PPA, and LLVM 6 from LLVM's PPA.

**Comment by ajor** [comment]
Fixed by #573
